sdk/src/main/java/io/horizen/account/proof/SignatureSecp256k1.java

```diff
@@ -2,86 +2,129 @@
 
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.fasterxml.jackson.annotation.JsonView;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
 import io.horizen.account.proposition.AddressProposition;
 import io.horizen.account.secret.PrivateKeySecp256k1;
 import io.horizen.account.utils.Secp256k1;
 import io.horizen.evm.Address;
+import io.horizen.json.serializer.HexNoPrefixBigIntegerSerializer;
 import io.horizen.proof.ProofOfKnowledge;
 import io.horizen.proof.ProofSerializer;
 import io.horizen.json.Views;
-import io.horizen.utils.BytesUtils;
+import java.math.BigInteger;
+
 
 @JsonView(Views.Default.class)
 public final class SignatureSecp256k1 implements ProofOfKnowledge<PrivateKeySecp256k1, AddressProposition> {
 
     @JsonProperty("v")
-    private final byte[] v;
+    @JsonSerialize(using = HexNoPrefixBigIntegerSerializer.class)
```

Comment:
As was discussed @JsonSerialize can be removed from this class

Reply:
I tried adding this in `ApplicationJsonSerializer` but it overrides the default behavior for other cases, for instance we have `value` and `nonce` of the `BigInteger` type in `EthereumTransaction`, therefore for the time being I left it as it is.

Reply:
good point, we for sure want other BigInteger values to be serialized in decimal form.